packages/bot-web-ui/src/components/quick-strategy/descriptions/strategy-description.tsx

```diff
@@ -1,56 +1,48 @@
 import React from 'react';
 import classNames from 'classnames';
 import { Text } from '@deriv/components';
-import { TDescriptionItem } from '../types';
+import { TStrategyDescription } from '../types';
 
-type TStrategyDescription = {
-    data: TDescriptionItem;
-    font_size: string;
-};
-
-const StrategyDescription: React.FC<TStrategyDescription> = ({ data, font_size }) => {
-    const descriptionContent = () => {
-        switch (data.type) {
-            case 'subtitle':
-                return data?.content?.map(text => (
-                    <div className='qs__long_description__title' key={text}>
-                        <Text size={font_size} weight='bold' dangerouslySetInnerHTML={{ __html: text }} />
-                    </div>
-                ));
-            case 'text': {
-                const class_names = classNames(`qs__long_description__content ${data?.className ?? ''}`);
-                return data?.content?.map(text => (
-                    <div className={class_names} key={text}>
-                        <Text size={font_size} dangerouslySetInnerHTML={{ __html: text }} />
-                    </div>
-                ));
-            }
-            case 'subtitle_italic':
-                return data?.content?.map(text => (
-                    <div className='qs__long_description__title italic' key={text}>
-                        <Text size={font_size} weight='bold' dangerouslySetInnerHTML={{ __html: text }} />
-                    </div>
-                ));
-            case 'text_italic': {
-                const class_names = classNames(`qs__long_description__content italic ${data?.className ?? ''}`);
-                return data?.content?.map(text => (
-                    <div className={class_names} key={text}>
-                        <Text size={font_size} dangerouslySetInnerHTML={{ __html: text }} />
-                    </div>
-                ));
-            }
-            case 'media':
-                return (
-                    <div>
-                        <img className='qs__long_description__image' src={data.src} alt={data.alt} />
-                    </div>
-                );
-            default:
-                return null;
+const StrategyDescription = ({ item, font_size }: TStrategyDescription) => {
+    const class_name = item?.className ?? '';
+    switch (item.type) {
+        case 'text': {
+            const class_names = classNames(`qs__description__content ${class_name}`);
+            return (
+                <>
+                    {item?.content?.map((text: string) => (
+                        <div className={class_names} key={text}>
+                            <Text size={font_size} dangerouslySetInnerHTML={{ __html: text }} />
+                        </div>
+                    ))}
+                </>
+            );
         }
-    };
-
-    return <div>{descriptionContent()}</div>;
```

Comment:
I rewrote the **invoke component** to a simple component because **we shouldn't invoke a component from the layout** for the following reasons:  
_1) Babel JSX transformer will transform it as a hook instead
2) Will be a problem if you are using conditional rendering, as hook order and count matter and it will throw a **Rendered fewer hooks than expected**_